js/job.jsx

```diff
@@ -118,13 +110,18 @@ export default class Job extends React.Component {
   }
 
   render() {
-    console.log('Beginning render()');
     const job = this.getJob();
-    if (!job) return null;
-    const jobConfCounter = this.state.jobConfs[this.props.params.jobId];
-    if (jobConfCounter === undefined) { return null; }
-    const {conf} = this.state.jobConfs[this.props.params.jobId];
+    if (!job) {
+      console.log('No job found');
+      return null;
+    }
     document.title = job.name;
+    const jobConf = this.state.jobConfs[this.props.params.jobId];
+    if (jobConf === undefined) {
+      console.log('No jobConf found');
+      return null;
```

Comment:
Looking at where `jobConf` is used, it does not appear fundamental to the page. Is it right to block the whole page from loading just because we don't have the conf yet?

Reply:
fixt!